skyportal/handlers/api/photometry.py

```diff
@@ -119,22 +121,41 @@ def post(self):
                               items:
                                 type: integer
                               description: List of new photometry IDs
+                            bulk_upload_id:
```

Comment:
For simplicity, why not always return a bulk-upload-id?  Otherwise the client has to add a special check.